src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure [] figures = new Figure[6];
+
+        for (int i = 0; i < 6; i++) {
```

Comment:
All magic numbers in your code should be constants. It's better to define the size of the figures array as a constant. For example, private static final int FIGURE_COUNT = 6; and then use this constant in the loop condition.